lib/daru/index/index.rb

```diff
@@ -222,6 +222,44 @@ def reorder(new_order)
       self.class.new(new_order.map { |i| from[i] })
     end
 
+    # Sorts a `Index` according to its values. Defaults to ascending order sorting.
+    # raises ArgumentError when all the values are not of the same class.
+    #
+    # == Options
+    #
+    # * +:ascending+ - if false, will sort in descending order. Defaults to true.
+    #
+    # * +:get_new_order+ - if true, then method will return the new order (in list) of
```

Comment:
I am using it [here](https://github.com/SciRuby/daru/pull/315/files/f6ce54c0c5a91b89526427ddfe8450f50c28949c#diff-a1adf8860e167030bf548e2529c7d508R249) . It is for getting the new arrangement order of index value. 

e.g if index is `[100, 99, 101, 1, 2]` then sorted will be `[1, 2, 99, 100, 101]` so the new arrangement of indexes is `[3, 4, 1, 0, 2]` I am calling it as `new_order`.